chore(doc): minor corrections in CONTRIBUTING (#397)
updatecli/updatecli: CONTRIBUTING.adoc

== REQUIREMENTS

> To build the project, just ensure to have access to the correct golang version and link:https://goreleaser.com/install/[goreleaser installed], then just run `make build`, or use Docker:

Reviewer: ```suggestion
To build the project, make sure that you link:golang[https://go.dev/] installed, link:https://www.gnu.org/software/make/[GNU Make] and link:https://goreleaser.com/install/[goreleaser installed], then execute the command `make build`, or use Docker with:
```
- Remove occurences of `just` (even though the litteral meaning is "only", it sometimes have a deameaning sense for beginners.
- Add reference to GNU make
-  Add link to golang website

```
docker run --rm -v "$PWD":/usr/src/updatecli -w /usr/src/updatecli -e GOOS=windows -e GOARCH=386 golang:1.16 go build -v
```

== CONTRIBUTE

> They are multiple ways to contribute which don't necessarily involve coding, like providing feedback, improving documentation, processes.

Reviewer: ```suggestion
They are multiple ways to contribute which don't necessarily involve coding, like providing feedback, improving documentation and processes.
```
